src/commands/setup.js

```diff
@@ -40,7 +40,7 @@ const fork = (org, repo, github_token) => {
 
 const submitPullRequest = (org, repo, projectPath, github_token) => {
   
-  let body = `This pull request adds backers and sponsors from your Open Collective https://opencollective.com/${repo} ❤️
+  let body = `Hi, I'm making updates for Open Collective. Either you or a supporter signed this repo up for open-collective. This pull request adds backers and sponsors from your Open Collective https://opencollective.com/${repo} ❤️
```

Comment:
no hyphen in "open collective" :). "Open Collective" would be even better. @monkeywithacupcake